synapse/lib/aha.py

```diff
@@ -537,41 +675,100 @@ def _initCellHttpApis(self):
         self.addHttpApi('/api/v1/aha/provision/service', AhaProvisionServiceV1, {'cell': self})
 
     async def initServiceRuntime(self):
+
         self.addActiveCoro(self._clearInactiveSessions)
 
         if self.isactive:
-            # bootstrap a CA for our aha:network
-            netw = self.conf.get('aha:network')
-            if netw is not None:
 
-                if self.certdir.getCaCertPath(netw) is None:
-                    logger.info(f'Adding CA certificate for {netw}')
-                    await self.genCaCert(netw)
+            # bootstrap a CA for our aha:network
+            netw = self.conf.req('aha:network')
 
-                name = self.conf.get('aha:name')
+            if self.certdir.getCaCertPath(netw) is None:
+                logger.info(f'Adding CA certificate for {netw}')
+                await self.genCaCert(netw)
 
+            name = self.conf.get('aha:name')
+            if name is not None:
                 host = f'{name}.{netw}'
                 if self.certdir.getHostCertPath(host) is None:
                     logger.info(f'Adding server certificate for {host}')
                     await self._genHostCert(host, signas=netw)
 
-                user = self._getAhaAdmin()
-                if user is not None:
-                    if self.certdir.getUserCertPath(user) is None:
-                        logger.info(f'Adding user certificate for {user}@{netw}')
-                        await self._genUserCert(user, signas=netw)
+            root = f'root@{netw}'
+            await self._genUserCert(root, signas=netw)
```

Comment:
Somethings fishy with this; I'm seeing it generating a new `root@{netw}` certificate on each boot of the service.

Reply:
Yea, i was just noticing that as well. I'll have a look :+1: